docs/reference/config-params.md

```diff
@@ -280,462 +324,543 @@ Otherwise it will default to `1`.
 
-### Possible options: shell command
+#### Possible options: shell command
 
-> kev.workload.liveness-probe-command:
+> x-k8s.workload.livenessProbe.exec.command
 ```yaml
 version: 3.7
 services:
   my-service:
-    labels:
-      kev.workload.liveness-probe-command: ["/is-my-service-alive.sh"]
+    x-k8s:
+    workload:
+      livenessProbe:
+        type: exec
+        exec:
+          command:
+            - /is-my-service-alive.sh
 ...
 ```
 
-## kev.workload.liveness-probe-http-port
+### x-k8s.workload.livenessProbe.http.port 
 
 Defines the liveness probe port to be used for the workload when the type is `http`. 
 See official K8s [documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-http-request). 
 
-### Possible options: Integer
+#### Possible options: Integer
 
-> kev.workload.liveness-probe-http-port:
+> x-k8s.workload.livenessProbe.http.port:
 ```yaml
 version: 3.7
 services:
   my-service:
-    labels:
-      kev.workload.liveness-probe-http-port: 8080
+    x-k8s:
+      workload:
+        livenessProbe:
+          type: http
+          http:
+            port: 8080
 ...
 ```
 
-## kev.workload.liveness-probe-http-path
+### x-k8s.workload.livenessProbe.http.path
 
 Defines the liveness probe path to be used for the workload when the type is `http`. 
 See official K8s [documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-http-request). 
 
-### Possible options: String
+#### Possible options: String
 
-> kev.workload.liveness-probe-http-path:
+> x-k8s.workload.livenessProbe.http.path:
 ```yaml
 version: 3.7
 services:
   my-service:
-    labels:
-      kev.workload.liveness-probe-http-path: /status
+    x-k8s:
+      workload:
+        livenessProbe:
+          type: http
+          http:
+            port: 8080
+            path: /status
 ...
 ```
 
-## kev.workload.liveness-probe-tcp-port
+### x-k8s.workload.livenessProbe.tcp.port
 
-Defines the liveness probe path to be used for the workload when the type is `tcp`. 
-See official K8s [documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-http-request). 
+Defines the liveness probe port to be used for the workload when the type is `tcp`.
+See official K8s [documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-tcp-liveness-probe).
 
-### Possible options: Integer
+#### Possible options: Integer
 
-> kev.workload.liveness-probe-tcp-port:
+> x-k8s.workload.livenessProbe.tcp.port:
 ```yaml
 version: 3.7
 services:
   my-service:
-    labels:
-      kev.workload.liveness-probe-tcp-port: 8080
+    x-k8s:
+      workload:
+        livenessProbe:
+          type: tcp
+          tcp:
+            port: 8080
 ...
 ```
 
-## kev.workload.liveness-probe-interval
```

Comment:
Was `interval` section accidentally removed?

Reply:
`interval` is now `period`.